.gitignore

```diff
@@ -56,6 +56,7 @@ scripts/install/config.sh
 /connect.lock
 /libpeerconnection.log
 /typings
+/.nx
```

Comment:
is it intended? or some left over config from your local env?

Reply:
This change is intended, not sure why, but some files in **.nx/cache** folder appears during the development.